.github/workflows/publish-pods.yml

```
@@ -86,13 +100,41 @@ jobs:
+            echo "🚀 Pushing Coralogix.podspec..."
+            if pod trunk push Coralogix.podspec --allow-warnings --verbose; then
+              echo "🎉 Coralogix.podspec pushed successfully! 🎉"
+            else
+              echo "⚠️ Push failed; re-checking for 409 duplicate..."
+              if pod trunk info Coralogix | grep -Eq "^[[:space:]]*-[[:space:]]*${TAG_RAW}([[:space:]]|\(|$)"; then
+                echo "ℹ️ Coralogix ${TAG_RAW} appears published. Done."
+              else
+                exit 1
+              fi
+            fi
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)